profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -109,7 +117,10 @@ fun ProfileScreen(
                             Spacer(Modifier.size(LocalPaddings.current.medium))
                             UserTabs(
                                 user = this@run,
-                                mediaCollection = viewerMediaLists?.data
+                                mediaCollection = viewerMediaLists?.data,
+                                onNavigateToMediaItem = onNavigateToMediaItem,
+                                sharedTransitionScope = sharedTransitionScope,
+                                animatedVisibilityScope = animatedVisibilityScope,
```

Comment:
don't shoot me if it's a completely whack idea, but what about shared transition scope as a CompositionLocal? Kinda goes against everything, but if we're gonna be using it everywhere... maybe we could assume it'll be there?

@boswelja thoughts?

Reply:
Shoot him

Just kidding :laughing: I haven't used shared element transitions enough to have an opinion, happy to give it a go